mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,17 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+     b has never been defined.
   2. Remove the line that throws the error.
+     console.log(b); has been removed.
   3. What is printed to the console?
+    2, 6, 4, 9, 6, 13, 8 
   4. How many times is "f1" called?
+    Three
   5. How many times is "f2" called?
```

Comment:
`f1` gets called each time `i` is an odd number, so I don't think 3 is the correct answer.